website/docs/components/form/primitives/partials/code/how-to-use.md

```diff
@@ -251,3 +246,7 @@ The layout of the content inside the "control" container is the responsibility o
   <F.Error>This is the error</F.Error>
 </Hds::Form::Fieldset>
 ```
+!!! callout
```

Comment:
Should this go underneath the `Form::Fieldset` heading? Similar to the one under `Form::Field`? Either way they should be the same.